package.json

```diff
@@ -57,8 +56,13 @@
     "redux"
   ],
   "dependencies": {
+    "broccoli-merge-trees": "^1.1.5",
     "ember-cli-babel": "^5.1.7"
   },
+  "peerDependencies": {
```

Comment:
guess replying to email doesn't attach to inline comment threads.  see my response below